packages/auth/src/accessToken/service.test.ts

```diff
@@ -73,8 +73,9 @@ describe('Access Token Service', (): void => {
   const BASE_ACCESS = {
     type: AccessType.OutgoingPayment,
     actions: [Action.Read, Action.Create],
+    identifier: v4(),
```

Comment:
This is supposed to be a url
https://github.com/interledger/open-payments/blob/main/auth-server-open-api-spec.yaml#L944-L946